sdk/identity/azure-identity/src/main/java/com/azure/identity/VisualStudioCodeCredential.java

```diff
@@ -31,7 +31,14 @@
  *
  * @see com.azure.identity
  * @see VisualStudioCodeCredentialBuilder
+ *
+ * @deprecated This credential is deprecated because the VS Code Azure Account extension on which this credential
+ * relies has been deprecated. Users should use other dev-time credentials, such as {@link AzureCliCredential},
+ * {@link AzureDeveloperCliCredential}, {@link AzurePowerShellCredential} or {@link IntelliJCredential} for their
+ * local development needs. See <a href="https://github.com/microsoft/vscode-azure-account/issues/964">here</a>
+ * for Azure account extension deprecation notice.
```

Comment:
```suggestion
 * @deprecated This credential is deprecated because the VS Code Azure Account extension on which this credential
 * relies has been deprecated. Users should use other dev-time credentials, such as {@link AzureCliCredential},
 * {@link AzureDeveloperCliCredential}, {@link AzurePowerShellCredential}, or {@link IntelliJCredential} for their
 * local development needs. See <a href="https://github.com/microsoft/vscode-azure-account/issues/964">this issue</a>
 * for Azure Account extension deprecation notice.
```